src/gmt_init.c

```diff
@@ -518,50 +522,80 @@ GMT_LOCAL void gmtinit_handle_escape_text (char *text, char key, int way) {
 	}
 }
 
+/*! . */
+GMT_LOCAL char * gmtinit_strchr_predexcl (char *string, char target, char predexcl) {
+	/* Version of strchr() that will ignore any instance of target
+	 * that is preceded by predexcl, the predecessor character
+	 * whose presence will exclude that instance from selection. */
+	char *c;
+	if (string == NULL) return NULL;
+	for (c = string; *c != '\0'; c++)
+		if ((*c == target) &&
```

Comment:
With our code style these two lines would be joined in one. We don't privilege the "vertical" styling.

Reply:
Which two lines exactly?

Reply:
I think @joa-quim means that the if-test on 533 should continue on that ine and not break at the &&, right?

Reply:
Yes, that's where I inserted the comment.

Reply:
OK. All right with all if I fix this in the next branch which I should be pushing later today? I'd prefer not to mess around in multiple branches on my local system, as I'm currently working on the next batch of updates for pswiggle (where I will incorporate this change).

Reply:
Sure, no problems. And you can even do it directly from the GH page.

Reply:
All right, now I see what you meant when you said that's where you inserted the comment. Still not clear to me how to change the code from that page so I'll just leave it alone until the next branch is done.

Reply:
Simple,
![image](https://user-images.githubusercontent.com/537321/180577937-4d24a18c-ad38-4e39-a6e1-e2f8527106fb.png)

Reply:
Thanks for the tip! (But will still wait for the next branch update.)

Reply:
Another tip here: you can just type `.` on this page, and you'll be redirected to the github.dev site, in which you can edit any files using the web-based vscode editor. I think it's the best editing experience using a web browser.

See https://github.com/github/dev for the official documentation.

Reply:
Thanks for that info!